specification/translation/Azure.AI.DocumentTranslation/examples/2023-11-01-preview/DocumentTranslate_MaximumSet_Gen.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "Translate a single document",
-  "operationId": "DocumentTranslationOperations_DocumentTranslate",
+  "operationId": "DocumentTranslationOperations_Translate",
```

Comment:
If you're not changing the operation name this change should probably be reverted?

Reply:
that's right. Updated